openapi.yaml

```diff
@@ -226,6 +226,50 @@ paths:
               schema:
                 $ref: '#/components/schemas/Problem'
 
+  /help:
+    get:
+      summary: Retrieve help information for the given command and any number of subcommands.
+      operationId: help
+      tags:
+        - help
+      parameters:
+        - name: command
+          in: query
+          style: form
+          explode: true
+          description: The command (and any subcommands) for which help information is needed. Subcommands can be separated by '/' or space ' '.
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Help information retrieved successfully.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+        "400":
```

Comment:
@princerajpoot20 we need to add `404` response too, because the command might not exist 👍. Also the error responses should always be `$ref: '#/components/schemas/Problem'`, we use this schema for any errors.